Day-2-Exercises/index.js

```diff
@@ -0,0 +1,102 @@
+let challange = '30 Days Of JavaScript'
+let corporations = 'Facebook, Google, Microsoft, Apple, IBM, Oracle, Amazon'
+let becauseCase = 'You cannot end a sentence with because because because is a conjunction'
+let trimCase = ' 30 Days Of JavaScript '
+
+
+//Exercise: Level 1
+
+console.log(challange)
+console.log(challange.length)
+console.log(challange.toUpperCase())
+console.log(challange.toLowerCase())
+console.log(challange.substring(3,7))
+console.log(challange.substring(3,challange.length))
+console.log('Check if there is a word Script:', challange.includes('Script'))
+console.log(challange.split())
+console.log(challange.split(' '))
+console.log(corporations.split(','))
+console.log(challange.replace('Java Script', 'Python'))
+console.log('Character at index 16 is:', challange.charAt('16'))
+console.log('Character code of J is:', challange.charCodeAt('11'))
+console.log('First occurrence of \'a\' is at:', challange.indexOf('a'), 'position')
+console.log('Last occurrence of \'a\' is at:', challange.lastIndexOf('a'), 'position')
+console.log('First occurrence of \'because\' is at:', becauseCase.indexOf('because'), 'position')
+console.log('Last occurrence of \'because\' is at:', becauseCase.lastIndexOf('because'), 'position')
+console.log('First occurrence of \'because\' is at:', becauseCase.search('because'), 'position')
+console.log(trimCase.trim())
+console.log(challange.startsWith('30'))
+console.log(challange.endsWith('JavaScript'))
+console.log(challange.match(/a/gi))
+console.log('30 Days of' + ' ' + 'JavaScript')
+console.log('30 Days of'.repeat(2))
+
+
+//Exercise: Level 2
+
+//Using console.log() print out the following statement:
+console.log('The quote \'There is no exercise better for the heart than reaching down and lifting people up.\' by John Holmes teaches us to help one another.')
+console.log('/"Love is not patronizing and charity isn\'t about pity, it is about love. Charity and love are the same -- with charity you give love, so don\'t just give money but reach out your hand instead.\"')
+
+//Check if typeof '10' is exactly equal to 10. If not make it exactly equal.
+console.log(typeof(parseInt('10')) == typeof(10))
+console.log(typeof('10') == typeof(toString(10)))
```

Comment:
There are two equality operators in JS: [normal ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality)(==) and [strict ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)(===). 
This task is about these operators. `typeof` operator always returns string value type.

Reply:
Updated in the code